packages/client/src/index.ts

```diff
@@ -32,20 +32,15 @@ const worldContract = new Contract(
     config.privateKey!,
     new ethers.providers.JsonRpcProvider(config.provider.jsonRpcUrl)
   )
-) as World;
+) as IWorld;
 
 // Just for demonstration purposes: we create a global function that can be
 // called to invoke the Increment system contract via the world. (See IncrementSystem.sol.)
 (window as any).increment = async () => {
-  const txResult = await worldContract["call"](
-    toBytes16("mud"),
-    toBytes16("increment"),
-    sigHash("increment()"),
-    {
-      gasPrice: 0,
-      gasLimit: 1_000_000,
-    }
-  );
+  const txResult = await worldContract.mud_increment_increment({
+    gasLimit: 1_000_000,
+    gasPrice: 0,
+  });
```

Comment:
unrelated to this PR but how can we remove the need for the manual gas setting?

Reply:
ethers seems to automatically set the gas price to non-zero if not overridden, but we could add a util to execute transactions (like a simple version of the old tx queue) which then handles the gas setting for users. The manual gas limit here is just for debugging (I needed the transaction to fail on the chain to get the revert reason instead of just "could not estimate gas").